base_layer/wallet/src/transaction_service/service.rs

```diff
@@ -1806,31 +1799,38 @@ where
 
     pub async fn register_code_template(
         &mut self,
-        fee_per_gram: MicroMinotari,
-        template_registration: CodeTemplateRegistration,
-        selection_criteria: UtxoSelectionCriteria,
-        message: String,
-        join_handles: &mut FuturesUnordered<
+        _fee_per_gram: MicroMinotari,
+        _template_name: String,
+        _template_version: u16,
+        _template_type: TemplateType,
+        _build_info: BuildInfo,
+        _binary_sha: FixedHash,
+        _binary_url: String,
+        _sidechain_deployment_key: Option<PrivateKey>,
+        _selection_criteria: UtxoSelectionCriteria,
+        _message: String,
+        _join_handles: &mut FuturesUnordered<
             JoinHandle<Result<TransactionSendResult, TransactionServiceProtocolError<TxId>>>,
         >,
-        transaction_broadcast_join_handles: &mut FuturesUnordered<
+        _transaction_broadcast_join_handles: &mut FuturesUnordered<
             JoinHandle<Result<TxId, TransactionServiceProtocolError<TxId>>>,
         >,
-        reply_channel: oneshot::Sender<Result<TransactionServiceResponse, TransactionServiceError>>,
+        _reply_channel: oneshot::Sender<Result<TransactionServiceResponse, TransactionServiceError>>,
     ) -> Result<(), TransactionServiceError> {
-        self.send_transaction(
-            self.resources.wallet_identity.address.clone(),
-            0.into(),
-            selection_criteria,
-            OutputFeatures::for_template_registration(template_registration),
-            fee_per_gram,
-            message,
-            TransactionMetadata::default(),
-            join_handles,
-            transaction_broadcast_join_handles,
-            reply_channel,
-        )
-        .await
+        todo!()
```

Comment:
Why is this function blocked out in this PR? I would expect some explanation in the PR description.

As an aside, I wonder what the 2nd layer will do with a lot of dust here, say 500,000 bogie registration UTXOs?